orthoinference/src/main/java/org/reactome/orthoinference/EventsInferrer.java

```diff
@@ -181,14 +181,14 @@ public static void inferEvents(Properties props, String pathToConfig, String spe
 			
 			try {
 				logger.info("Attempting to infer " + reactionInst);
-				ReactionInferrer.inferReaction(reactionInst);
+				ReactionInferrer.inferReaction(reactionInst, threshold);
```

Comment:
I think rather than including the threshold as an argument I'd prefer it set as a global variable in the necessary classes (see `setSummationInstance()` or `setInstanceEdits` in this class) . That's the way I've done it with other, similar values and I'd like to keep things uniform

Reply:
I think that's a good idea.  If we could localize it to one place and reference it from there by parts of the code that need it, it would be handy.

Reply:
Agreed. It was a pain to thread it through all of those static methods. I had looked for a good place to set it gloabally, but I would like it to exist outside of the compiled code if possible. We might need to adjust this value in the future.

Reply:
For sure, the value should live outside the compiled code.  The threshold could be declared in one place, initialized by parsing the configuration, and referenced where needed by the code from the single field containing it.

Reply:
on it.

Reply:
done. in config.properties.